account_followup/models/res_partner.py

```diff
@@ -0,0 +1,447 @@
+        copy=False,
+        help="Optionally you can assign a user to this field, which will "
+             "make him responsible for the action.",
+    )
+    payment_note = fields.Text(
+        string='Customer Payment Promise',
+        help="Payment Note",
+        track_visibility="onchange",
+        copy=False,
+    )
+    payment_next_action = fields.Text(
+        string='Next Action',
+        copy=False,
+        track_visibility="onchange",
+        help="This is the next action to be taken.  It will automatically "
+             "be set when the partner gets a follow-up level that requires "
+             "a manual action. ",
+    )
+    payment_next_action_date = fields.Date(
+        string='Next Action Date',
+        copy=False,
+        help="This is when the manual follow-up is needed. "
+             "The date will be set to the current date when the partner "
+             "gets a follow-up level that requires a manual action. "
+             "Can be practical to set manually e.g. to see if he keeps "
+             "his promises.",
+    )
+    unreconciled_aml_ids = fields.One2many(
+        comodel_name='account.move.line',
+        inverse_name='followup_id',
+        string='partner_id',
+        domain=[('reconciled', '=', False)],
+    )
+    latest_followup_date = fields.Date(
+        string="Latest Follow-up Date",
+        store=False,
+        compute='_compute_latest',
+        help="Latest date that the follow-up level of the partner was changed",
+    )
+    latest_followup_level_id = fields.Many2one(
+        comodel_name='account_followup.followup.line',
+        string="Latest Follow-up Level",
+        help="The maximum follow-up level",
+        compute='_compute_latest',
+    )
+    latest_followup_level_id_without_lit = fields.Many2one(
+        comodel_name='account_followup.followup.line',
+        string="Latest Follow-up Level without litigation",
+        help="The maximum follow-up level without taking into account the "
+             "account move lines with litigation",
+        compute='_compute_latest',
+    )
+    payment_amount_due = fields.Float(
+        string="Amount Due",
+        compute='_compute_amounts_and_date',
+    )
+    payment_amount_overdue = fields.Float(
+        string="Amount Overdue",
+        compute='_compute_amounts_and_date',
+    )
+    payment_earliest_due_date = fields.Date(
+        string="Worst Due Date",
+        compute='_compute_amounts_and_date',
+    )
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type=None, toolbar=False,
+                        submenu=False):
+        res = super(ResPartner, self).fields_view_get(
+            view_id=view_id, view_type=view_type, toolbar=toolbar,
+            submenu=submenu)
+        if view_type == 'form' and self.env.context.get('Followupfirst'):
+            doc = etree.XML(res['arch'], parser=None, base_url=None)
+            first_node = doc.xpath("//page[@name='followup_tab']")
```

Comment:
This piece of code open payment followup tab first in partner form, when you are doing a manual follow-up. When you open the partner form in other cases, payment followup tab is in other position, and first tab is open is "Contacts & addresses"